src/app/frontend/deploy/namespace_dialog.html

```diff
@@ -0,0 +1,33 @@
+<!--
```

Comment:
Completely agree, now it looks a look better than when it was `namespace_dialog`.